client/src/App.jsx

```diff
@@ -1,27 +1,12 @@
-import logo from "./logo.svg";
-import "./App.css";
+import Create from "./components/Create";
+import Edit from "./components/Edit";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-          {}
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
```

Comment:
This is an unnecessary tag

Reply:
You can't render two components without a fragment, so I guess for now it may be necessary.